src/error.rs

```diff
@@ -31,4 +33,13 @@ pub enum Error {
     InvalidAttribute(String),
     #[error("invalid axis: {0}")]
     InvalidAxis(String),
+    #[error("deserialization error: {0}")]
+    DeserializationError(String),
```

Comment:
Should be a `SerdeError` instead of a `String`?

Reply:
Yes